packages/arb-token-bridge-ui/src/util/rpc/getRpcUrl.ts

```diff
@@ -0,0 +1,36 @@
+import { ChainId } from '../../types/ChainId'
+
+import { getInfuraRpcUrl } from './infura'
+import { getAlchemyRpcUrl } from './alchemy'
+
+type RpcProvider = 'infura' | 'alchemy'
+
+export type ProductionChainId = Exclude<
+  ChainId,
+  ChainId.Local | ChainId.ArbitrumLocal | ChainId.L3Local
+>
+
+function getRpcProvider(): RpcProvider {
+  const rpcProviderFromEnv = process.env.NEXT_PUBLIC_RPC_PROVIDER
+  return (rpcProviderFromEnv?.toLowerCase() as RpcProvider) || 'infura' // the || is intentional to handle empty strings
```

Comment:
instead of a direct type cast, we can check if `rpcProviderFromEnv` is `infura` or `alchemy`, if so return it, otherwise return `infura`?